java/src/jmri/script/ScriptEngineSelector.java

```diff
@@ -37,6 +37,10 @@ public Engine getSelectedEngine() {
      */
     public void setSelectedEngine(@Nonnull Engine engine) {
         _selectedEngine = engine;
+        if (engine.getLanguageName().equals(ECMA_SCRIPT)) {
+            log.warn("*** Scripting with JavaScript/ECMAscript is being deprecated.***");
+            log.warn("*** Please contact us on the jmriusers group for assistance. ***");
+        }
```

Comment:
For me as a developer, it's very clear what this is about. But for a user, it might be confusing or seem to be irrelevant. I suggest adding: `and is about to be removed`.

`*** Scripting with JavaScript/ECMAscript is being deprecated and is about to be removed.***`